seeds/userIndustry.js

```diff
@@ -0,0 +1,67 @@
+const { UserIndustry } = require('../models');
+
+const userIndustryData = [
+    {
+        
+        user_id: 1,
+        industry_id: 1
+    },
+
+    {
+        user_id: 1,
+        industry_id: 2
+    },
+
+    {
+        
+        user_id: 2,
+        industry_id: 2
+    },
+
+    {
+        
+        user_id: 2,
+        industry_id: 4
+    },
+
+    {
+        user_id: 3,
+        industry_id: 3
+    },
+
+    {
+        user_id: 3,
+        industry_id: 1
+    },
+
+    {
+        
+        user_id: 4,
+        industry_id: 6
```

Comment:
```suggestion
        industry_id: 4 
```

This is an invalid industry ID of 6 as we have only defined the seeds for 5 industries.